test/soak/cauldron/cauldron-rc.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: cauldron
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    app: cauldron
+  template:
+    metadata:
+      labels:
+        app: cauldron
+    spec:
+      containers:
+      - name: cauldron
+        image: kubernetes/cauldron:1.0
+        imagePullPolicy: Always
+        env:
+        - name: "up_to"
```

Comment:
Added missing `--` and it now works. I've switched the cauldron binary to run once by default which is what I want and for the RC config to make it run indefinitely. Thanks. PTAL -- I hope it is ready to merge now.